chronosphere/tfschema/notification_policy.go

```diff
@@ -85,6 +85,11 @@ var NotificationRouteSchema = typeset.Set{
 		"repeat_interval": Duration{
 			Optional: true,
 		},
+		"group_by": typeset.NotNormalized(&schema.Schema{
```

Comment:
we are modelling group_by differently in TF and the API.
In the API, it looks like group_by is a nested struct, which has a LabelNames list of strings
in TF, we have group_by directly be a list of string

For older fields, there may be mismatches because we cleaned up the API but left TF for backwards compatibility, but for new fields, we'd like the TF and API to be consistent

Reply:
thanks for the comment, done!